mkdocs.yml

```diff
@@ -24,6 +24,7 @@ nav:
               - Connecting Cloud Storage: advanced-usage/connecting-cloud-storage/index.md
               - S3-Compatible APIs: advanced-usage/connecting-cloud-storage/s3-compatible.md
               - GCP: advanced-usage/connecting-cloud-storage/gcp.md
```

Comment:
Not added in this PR, but I missed it first time around — is there a better (more complete) name than `GCP` for this integration?

Reply:
Any of `Google Cloud Platform` / `Google Cloud` / `Google Cloud Storage` could be considered complete; any preference of those three? (I'd like to leave the page file name as it's already linked from the app)

Reply:
I think my personal preference would be `Google Cloud Storage` as this is most specifically what we are connecting to